Detections/Ubiquiti/UbiquitiCryptominer.yaml

```diff
@@ -0,0 +1,27 @@
+id: 7feb3c32-2a11-4eb8-a2d7-e3792b31cb80
+name: Ubiquiti - Possible connection to cryptominning pool
+description: |
+  'Detects connections which may indicate that device is infected with cryptominer.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Ubiquiti
+    dataTypes:
+      - Ubiquiti_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+query: |
+  let susp_dst_ports = dynamic(['14433', '14444', '3333', '3334', '3335', '3336', '4444', '45560', '45700', '5555', '5556', '6666', '7777', '8788', '8888', '8899', '9999', '10034']);
+  UbiquitiAuditEvent
+  | where ipv4_is_private(SrcIpAddr)
+  | where ipv4_is_private(DstIpAddr) == 'False'
+  | where DstPortNumber in (susp_dst_ports)  or (DnsQuery contains 'pool' and DnsQuery contains 'xmr')

```

Comment:
Don't recommend using contains here unless it is absolutely necessary.  Try using ```has```

Reply:
in this particular case it is absolutely necessary to use 'contains'